[Alerts][Doc] Added README documentation for API key invalidation configuration options. (#92757)
elastic/kibana: x-pack/plugins/alerts/README.md

2. Configure feature level privileges using RBAC 
3. Create an alert using the RESTful API [Documentation](https://www.elastic.co/guide/en/kibana/master/alerts-api-update.html) (see alerts -> create).

## Alerts API keys

> When we create an alerts we generate a new API key.

Reviewer: ```suggestion
When we create an alert, we generate a new API key.
```

> When we change, delete or manage alert availabilty we need to invalidate the old API key and create a new one (except the delete process).

Reviewer 2: If we're not doing it for delete, should we leave it out of the list? `When we update, enable or disable an alert, we need to invalidate the old API key and create a new one`

Reviewer: ```suggestion
When we update, enable, or disable an alert, we must invalidate the old API key and create a new one.
```

Author: I will try try to rephrase it, because we are doing invalidation for DELETE, but not generate a new api key.

> For managing API keys invalidation process we use additional saved object `api_key_pending_invalidation`, which stores all alerts API keys that was marked for invalidation during the updating or changing alerts.

Reviewer: ```suggestion
To manage the invalidation process for API keys, we use the saved object `api_key_pending_invalidation`.  This object stores all API keys that were marked for invalidation when alerts were updated.
```

> For executing security plugin invalidation we schedule a task which runs by default for every 5 mins to check if `api_key_pending_invalidation` saved object contains new API keys which marked for invalidation earlier then configured delay: default value is 5 mins.

Reviewer: ```suggestion
For security plugin invalidation, we schedule a task to check if the`api_key_pending_invalidation` saved object contains new API keys that are marked for invalidation earlier than the configured delay.  The default value for running the task is 5 mins.
```

> To change the default invalidation task schedule, use kibana.yml configuration option `xpack.alerts.invalidateApiKeysTask.interval: '5m'`.

Reviewer: ```suggestion
To change the schedule for the invalidation task, use the kibana.yml configuration option `xpack.alerts.invalidateApiKeysTask.interval`.
```

> To change the default delay for the makrked API key invalidation, use kibana.yml configuration option `xpack.alerts.invalidateApiKeysTask.removalDelay: '5m'`.

Reviewer: ```suggestion
To change the default delay for the API key invalidation, use the kibana.yml configuration option `xpack.alerts.invalidateApiKeysTask.removalDelay`.
```
